radix-engine-interface/src/data/value_serializer.rs

```diff
@@ -527,7 +527,7 @@ pub fn serialize_custom_value<S: Serializer>(
             serializer,
             context,
             ScryptoCustomValueKind::NonFungibleLocalId,
-            &value.serializable(*context),
+            &format!("{}", value),
```

Comment:
We need to remove `ContextualSerialize<ScryptoValueFormattingContext<'a>> for NonFungibleLocalId` and `ContextualSerialize<ScryptoValueFormattingContext<'a>> for NonFungibleGlobalId` below I think?

Reply:
And to update `ScryptoCustomValue::NonFungibleGlobalId` to use `format!` as well?

Reply:
`NonFungibleGlobalId` still needs context for the resource address contained.